apps/core-v2-testnet/typechain-types/Greeter.ts

```diff
@@ -0,0 +1,120 @@
+/* Autogenerated file. Do not edit manually. */
+/* tslint:disable */
+/* eslint-disable */
+import type {
+  BaseContract,
+  BigNumber,
+  BytesLike,
+  CallOverrides,
+  ContractTransaction,
+  Overrides,
+  PopulatedTransaction,
+  Signer,
+  utils,
+} from "ethers";
+import type { FunctionFragment, Result } from "@ethersproject/abi";
+import type { Listener, Provider } from "@ethersproject/providers";
+import type {
+  TypedEventFilter,
+  TypedEvent,
+  TypedListener,
+  OnEvent,
+  PromiseOrValue,
+} from "./common";
+
+export interface GreeterInterface extends utils.Interface {
```

Comment:
We should pull the typechain stuff out to its own `packages/core-v2-typechain` and have core-v2-testnet depend on it-- we'll want to re-use the typechain stuff in a few places and it won't make sense to import them from the testnet in all cases.

Reply:
The v2 contracts won't live in this repo. This dir was just generated by the default hardhat project example. It would be helpful for me to have some example contracts here when testing util functions I add before v2 contracts. Thoughts?